src/pages/[platform]/build-a-backend/storage/authorization/index.mdx

```diff
@@ -177,9 +187,9 @@ When a function is granted access to storage, it also receives an environment va
 There are some limitations on the types of prefixes that can be specified in the storage access definition.
 
 1. All paths are treated as prefixes from the storage root. As such all paths must end with `/*` to make this explicit.
-2. Only one level of nesting is allowed. For example, you can define access controls on `foo/*` and `foo/bar/*` but not on `foo/bar/baz/*` because that path has 2 other prefixes.
-3. Wildcards cannot conflict with the `{entity_id}` token. For example, you cannot have both `foo/*` and `foo/{entity_id}/*` defined because the wildcard in the first path conflicts with the `{entity_id}` token in the second path.
-4. A path cannot be a prefix of another path with an `{entity_id}` token. For example `foo/*` and `foo/bar/{entity_id}/*` is not allowed.
+2. Only one level of nesting is allowed. For example, you can define access controls on `folder/*` and `folder/documents/*` but not on `folder/documents/photos/*` because that path has 2 other prefixes.
+3. Wildcards cannot conflict with the `{entity_id}` token. For example, you cannot have both `folder/*` and `folder/{entity_id}/*` defined because the wildcard in the first path conflicts with the `{entity_id}` token in the second path.
+4. A path cannot be a prefix of another path with an `{entity_id}` token. For example `folder/*` and `folder/documents/{entity_id}/*` is not allowed.
 
 ### Prefix behavior
```

Comment:
I don't have a good alternative. But this has actually less to do with the prefix behavior and more to do with the authorization rules' evaluation order